README.md

```diff
@@ -1,28 +1,26 @@
-# Kosaksi-Pasapugazh-and-experiments# KosaksipasapugazhExperiments
+# KosaksipasapugazhExperiments 🚀
```

Comment:
Use proper format for title. Use spaces in between words and capitalize first letter.